src/pages/q/[[...all]].tsx

```diff
@@ -88,6 +92,20 @@ const SearchIndex: any = ({
     searchState,
   )
 
+  if (error) {
+    return (
+      <div className="h-screen flex items-center justify-center gap-4">
+        <Image
+          src="https://res.cloudinary.com/dg3gyk0gu/image/upload/v1659039546/eggodex/basic_eggo.png"
+          alt="egghead search error"
+          width={200}
+          height={200}
+        />
+        <p className="prose dark:prose-dark text-xl">{error}</p>
+      </div>
+    )
+  }
```

Comment:
if we return an error from `getServerSideProps` we display it. Currently just returning `'Rate limit exceeded, please try again in 60 seconds.'`

along with eggo ![eggo says stop](https://res.cloudinary.com/dg3gyk0gu/image/upload/v1659039546/eggodex/basic_eggo.png)